libvcx/src/v3/handlers/connection/state_machine_invitee.rs

```diff
@@ -0,0 +1,549 @@
+use std::collections::HashMap;
```

Comment:
It would be nice to also split `messages.rs` and move inviter / invitee and their SMs into separate dirs, similar to the other protocols (presentation / issuance)

Reply:
Agree, I'll do this next step